src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,146 @@
-import org.assertj.core.api.Assertions;
-import org.openqa.selenium.By;
+import org.assertj.core.api.SoftAssertions;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.Homepage;
 import utils.WebDriverUtils;
 
-import java.time.Duration;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class HomepageTest {
 
     private WebDriver driver;
 
+    private static final String WHITE_COLOR = "#ffffff";
+    private static final String BLUE_COLOR = "#1979c3";
+
     @BeforeClass
     public void setUp() {
         driver = WebDriverUtils.createWebDriver();
     }
 
+    @BeforeMethod
+    public void beforeMethod() {
+        driver.get("https://magento.softwaretestingboard.com/");
+    }
+
     @Test
     public void homepage_MainBanner_HasTextAndButtonCorrectlyDisplayed() {
         //given
         Homepage homepage = new Homepage(driver);
 
         //then
-        assertThat(homepage.getMainBannerTopText()).isEqualTo("New Luma Yoga Collection");
-        assertThat(homepage.getMainBannerMainText()).isEqualTo("Get fit and look fab in new seasonal styles");
-        assertThat(homepage.getMainBannerButtonText()).isEqualTo("Shop New Yoga");
-        assertThat(driver.findElement(By.cssSelector(".blocks-promo > a")).getAttribute("href")).
-                isEqualTo("https://magento.softwaretestingboard.com/collections/yoga-new.html");
-        assertThat(driver.findElement(By.cssSelector(".blocks-promo > a img")).getAttribute("src")).
-                isEqualTo("https://magento.softwaretestingboard.com/pub/media/wysiwyg/home/home-main.jpg");
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isMainBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isMainBannerButtonEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getMainBannerTopText()).isEqualTo("New Luma Yoga Collection");
+        softAssertions.assertThat(homepage.getMainBannerMainText()).isEqualTo("Get fit and look fab in new seasonal styles");
+        softAssertions.assertThat(homepage.getMainBannerButtonText()).isEqualTo("Shop New Yoga");
+        softAssertions.assertThat(homepage.getMainBannerLink()).isEqualTo("https://magento.softwaretestingboard.com/collections/yoga-new.html");
```

Comment:
https://magento.softwaretestingboard.com/ is a part of each expected image link, href link, source link etc.
And it's used many times on this class.

It should be extracted to constant and then first used in the setUp() method, then in all places when urls contains them.

softAssertions.assertThat(homepage.getMainBannerLink()).isEqualTo(HOMEPAGE_URL + "collections/yoga-new.html");